exporter/sentryexporter/config.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 OpenTelemetry Authors
```

Comment:
Better drop the dates:

```suggestion
// Copyright The OpenTelemetry Authors
```

Reply:
This is the format they use in the repository - https://github.com/open-telemetry/opentelemetry-collector-contrib/search?q=copyright+2020&unscoped_q=copyright+2020

Reply:
Will change to `Copyright 2020, OpenTelemetry Authors`. Does that seem like a better fit to you?

Reply:
This came up in other repos, the direction seems to not have dates anymore. Let me find links

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/964